apps/SageAccountWeb/src/components/ProfileValidation/ReturnToAppButton.tsx

```diff
@@ -11,21 +11,17 @@ import {
   useTheme,
 } from '@mui/material'
 import CloseIcon from '@mui/icons-material/Close'
-import { useAppContext } from '../../AppContext'
 import React from 'react'
 import { NavLink } from 'react-router-dom'
 
 export const ReturnToAppButton: React.FC<{ children?: React.ReactNode }> = ({
   children,
 }) => {
-  const { redirectURL } = useAppContext()
   const theme = useTheme()
   const [open, setOpen] = React.useState(false)
   const onClose = () => setOpen(false)
   const redirectFn = () => {
-    redirectURL
-      ? window.location.assign(redirectURL)
-      : window.location.assign('/authenticated/myaccount')
+    window.location.assign('/authenticated/myaccount')
```

Comment:
https://sagebionetworks.jira.com/browse/PORTALS-3104

Reply:
Could also use react router `history.push` since we're staying in the app

Reply:
That seems reasonable